### DISCUSSION TITLE
Sudden weird styling of popups & dropdowns

### DISCUSSION THREAD

**Discussion by PepijnVDK1** [filamentphp/filament#12240] [Help] [answered]
### Package

Other

### Package Version

v3.1

### How can we help you?

Hi, 
I don't know whats suddenly causing this. I've been trying to revert all commits to see where the issue lies.
I've noticed that popups and dropdowns suddenly started to stretch full page and out of the viewport.

Has anyone had any similar issues with this? 
<img width="1038" alt="Screenshot 2024-04-08 at 08 24 40" src="https://github.com/filamentphp/filament/assets/73099380/32a877d4-a8bf-472c-b1b6-7f599844b47c">
<img width="1062" alt="Screenshot 2024-04-08 at 08 25 32" src="https://github.com/filamentphp/filament/assets/73099380/f86c0ab8-e1e5-4eb9-b2ad-69185263edc8">

Kind regards.

**Comment by leandrocfe** [comment]
run `npm run build`

**Reply by PepijnVDK1** [reply]
Hi, thanks for your answser.

Already did that. It does not resolve the issue unfortunately

**Reply by leandrocfe** [reply, marked as answer]
`php artisan filament:upgrade`
?

**Reply by PepijnVDK1** [reply]
Oh wow, 
🤦‍♂️ That did the trick!

Thank you so much!
